lib/elix/brain/redis_store.ex

```diff
@@ -0,0 +1,65 @@
+defmodule Elix.Brain.RedisStore do
+  @moduledoc """
+  A Redis back-end for the Brain which persists memory to disk.
+  """
+
+  @behaviour Elix.Brain.Store
+  @encoded_binary_prefix "binterm:"
+
+  def start_link do
+    redis_url = System.get_env("REDIS_URL") || "redis://"
```

Comment:
Why's that? To prevent people from using a Redis instance they may not intend? I generally agree about having a higher bar for intentionality in production.